seaworthy/lowlevel.py

```diff
@@ -0,0 +1,59 @@
+"""
+This is a module for horrible low-level things that we really wish we
+didn't have to build and maintain ourselves.
+"""
+
+import select
```

Comment:
Do we want to use [`selectors`](https://docs.python.org/3/library/selectors.html)?